app/javascript/dashboard/routes/dashboard/settings/inbox/channels/Twilio.vue

```diff
@@ -17,6 +17,34 @@
     </div>
 
     <div class="medium-8 columns">
+      <input
+        id="useMessagingService"
+        v-model="useMessagingService"
+        type="checkbox"
```

Comment:
What's the best way to test these and the other frontend changes?

Reply:
We don't have many specs for the presentation layer. We use jest to test the stores and related business logic. 

Here is one example you can take a look at.
https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/dashboard/components/layout/specs/AvailabilityStatus.spec.js

Skipping this should be fine for now.

Reply:
> Skipping this should be fine for now.

Sounds good 👍